routes/api/auth.js

```diff
@@ -0,0 +1,38 @@
+const router = require('express').Router();
+const config = require('./../../config');
+const jwt = require('jsonwebtoken');
+const { crudService, errorService } = require('./../../services');
+const { validationMiddleware } = require('./../../middlewares');
+const { UserModel } = require('./../../models');
+const { SCHEMAS, SCHEMAS_TYPE } = require('./../../shared/constants');
+
+router.post(
+    '/login',
+    validationMiddleware(SCHEMAS.USER, SCHEMAS_TYPE.READ),
+    (req, res) => {
+        const errorMsg = 'Invalid Password / User Name';
+
+        crudService
+            .findByKey(UserModel, res, { firstName: req.body.firstName} , false)
```

Comment:
just for now its ok to use firstName to find user upon login, but better use email or username, which should be unique, otherwise its possible to issue when there are two or more user with the same "firstName"

Reply:
OK. I will change it and add email field to User Model.